init_docs/Text.json

```diff
@@ -939,5 +939,6 @@
     "Course_Answers_Replicated":"Course Answers Replicated",
     "Please_provide_atleast_one_correct_answer":"Please provide atleast one correct answer",
     "You_choose_the_option_without_correct_answer":"You choose the option without correct answer",
-    "Auto_Publication":"Auto Publication"
+    "Auto_Publication":"Auto Publication",
+    "Resign_Publication":"Resign Publication"
```

Comment:
Resign Publication does not sound good.
It could be 'Enable Auto Sync' and 'Disable Auto Sync' or something like that.